src/frontend/components/PriorityPalette/PriorityPalette.js

```diff
@@ -0,0 +1,46 @@
+import "./PriorityPalette.css";
+
+const PriorityPalette = ({ userData, setUserData }) => {
+  const handlePriority = (priority) => {
+    setUserData({ ...userData, priority: priority });
+  };
+  return (
+    <div className="prioritypall">
+      <div className="priority__list">
+        <input
+          type="radio"
+          name="priority"
+          className="priority__input"
+          value="1"
+          checked={userData.priority.Low === "1"}
+          onChange={(e) => handlePriority({ Low: e.target.value })}
+        ></input>
+        <label>Low</label>
```

Comment:
You can wrap the `input` field inside the `label` or provide a `for` attribute to the label so that the radio buttons gets selected if clicked on the label.